src/app/donation-start/donation-start-form/donation-start-form-new.component.html

```diff
@@ -139,9 +139,9 @@ <h2 class="b-rt-0 b-m-0 b-bold">Tip Big Give</h2>
           <mat-expansion-panel (opened)="panelOpenState = true"
                                (closed)="panelOpenState = false">
             <mat-expansion-panel-header>
-              <h3 class="b-rt-0 b-m-0 b-bold">How does Big Give use tips?</h3>
+              <h3 class="b-rt-0 b-m-0 b-bold"><span class="span-hr"></span>How does Big Give use tips?</h3>
```

Comment:
Couldn't we apply a border below the `h3` itself? This feels like it would be easier to follow and less brittle than adding an absolutely positioned extra element.

Reply:
Hey @NoelLH , yes I remember trying that but couldn't make the border extend to the edges this way
![Screenshot from 2023-07-18 12-16-41](https://github.com/thebiggive/donate-frontend/assets/57061244/a938129f-0a93-4285-9172-95006a8ee464)

Reply:
@dorota-joanna OK. I think let's not spend more time on this part as we can hopefully fix the components version of the accordion at some point and substitute it in.